build/dockerfiles/rhel.Dockerfile

```diff
@@ -39,7 +39,7 @@ ENV LATEST_ONLY=${LATEST_ONLY}
 
 # NOTE: uncomment for local build. Must also set full registry path in FROM to registry.redhat.io or registry.access.redhat.com
 # enable rhel 7 or 8 content sets (from Brew) to resolve jq as rpm
-COPY ./build/dockerfiles/content_sets_epel7.repo /etc/yum.repos.d/
+COPY ./build/dockerfiles/content_sets_centos8_appstream.repo /etc/yum.repos.d/
```

Comment:
DUDE. You got Centos in my RHEL. Not cool, bro.

Reply:
Give me a RHEL repo that contains skopeo and doesn't require a subscription and I'll toss it in here :)

Reply:
well, I need to find one for this to work downstream because Brew can't see epel and centos repos so ... challenge accepted & required.

Reply:
skopeo is in the appstream repo for RHEL8 so to build this locally we have three options:
* RHEL subscription
* use Brew internal pulp .repo
* use centos public .repo
 
Though I really thought that rpms in UBI were also free w/o subscription. Hmm. Maybe not?